### ISSUE TITLE
feature-visualization

### ISSUE THREAD

**Issue by tearhupo121031** [tensorflow/lucid#78]
Sorrry for bothering,
I try to do the feature-visualization for VGG16 in pytorch, but the result of my picture lost much of color information, some results are as follows:
![feature-visualization](https://user-images.githubusercontent.com/28970557/42791756-cf61460c-89a4-11e8-907f-1d3e3dc684f2.jpg)

my loss function is the mean of the output, do I need any other constraints？

**Comment by ludwigschubert** [comment]
Hi @tearhupo121031 , we discuss various regularization terms and show some of their effects in [this section of Feature Visualization](https://distill.pub/2017/feature-visualization/#enemy-of-feature-vis). I'm afraid I can't be of more help—all of this is research territory, so we don't yet have good rules of thumb either.
